cstrml/tee/src/lib.rs

```diff
@@ -299,18 +299,34 @@ impl<T: Trait> Module<T> {
             // a. calculate this controller's order file map
             // TC = O(nm), `n` is stored files number and `m` is corresponding order ids
             let mut order_files: Vec<(MerkleRoot, T::Hash)> = vec![];
+            let mut punishment_order_ids: Vec<T::Hash> = vec![];
+            let mut overdue_order_ids: Vec<T::Hash> = vec![];
             if let Some(provision) = T::MarketInterface::providers(&controller) {
                 for (f_id, order_ids) in provision.file_map.iter() {
                     for order_id in order_ids {
+                        
                         // Get order status(should exist) and (maybe) change the status
                         let sorder =
                             T::MarketInterface::maybe_get_sorder(order_id).unwrap_or_default();
                         if sorder.status == OrderStatus::Success {
                             order_files.push((f_id.clone(), order_id.clone()))
                         }
+                        if sorder.completed_on >= Self::get_current_block_number() {
```

Comment:
need a confirmation here. closing a sorder would delete file_map. Don't know the real behavior.

Reply:
Is this issue still be?

Reply:
`order_files` to `success_order_files`

Reply:
add `TODO: add punishment logic when we close overdue sorder`